### DISCUSSION TITLE
java8/java14

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#266] [Announcements]
# java8/java14

本文详细介绍了Java 14的新特性，包括Record、instanceof和switch语句的改进。通过实际的代码示例，展示了如何使用这些新特性来简化代码，提高编程效率。学习本文，让您快速了解Java 14的新亮点，领略新特性带来的编程魅力。

https://javabetter.cn/java8/java14.html

**Comment by absoultly-classical** [comment]
这文本块就是python的语法
